### PR TITLE
Improvements to readme: install instructions & running the service

### PR DESCRIPTION
**Pull request by aepete**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

1. Improves installation instructions:
    - Order of installation updated to be consistent with what's described in the [Getting Started document](https://nio365.sharepoint.com/sites/ETWMFGlobalCollaboration/Shared%20Documents/Forms/AllItems.aspx?csf=1&web=1&e=z2syht&siteid=%7BC774B937%2DCE0B%2D421A%2DACA2%2DE73E7769CD91%7D&webid=%7BEE89A1E8%2D32EE%2D40DC%2DB09F%2D3BAE307F8F4C%7D&uniqueid=%7BFA4ABCC5%2D1404%2D42A3%2DBB40%2D2C404114B385%7D&FolderCTID=0x0120000580FD3CAC7FFF439CBF9AB4BD0CE348&id=%2Fsites%2FETWMFGlobalCollaboration%2FShared%20Documents%2FDocs%20and%20Demos%2FDocumentation%20%28Download%20This%20Folder%29%2Fmeasurementlink%5Fgetting%5Fstarted%5F1%2D31%2D2023%5FDRAFT%2Epdf&viewid=cf518915%2De0e9%2D418c%2D981d%2D917b9933d454&parent=%2Fsites%2FETWMFGlobalCollaboration%2FShared%20Documents%2FDocs%20and%20Demos%2FDocumentation%20%28Download%20This%20Folder%29): InstrumentStudio, MeasurementLink, then the measurement ADE (in this case, LabVIEW).
    - Replaces listing of supported LV versions with simply listing the minimum version required.  This is consistent with the getting started guide and grpc-labview's approach to specifying supported versions.
2. Subtle technical improvement to the instructions for running a measurement, reflecting the fact that the `STOP` button now appears on the window that pops up after having started the service, rather than the top-level `Run Service.vi` itself.  Supplemented the updated instruction with an image.
3. Cleans up quick-links in the document's header; some were stale from the corresponding sections having been removed or renamed.

### Why should this Pull Request be merged?

Makes the documentation more thorough, correct, complete, and consistent with other docs.

### What testing has been done?

- Rendered page appears correct.
- Confirmed with @rfriedma that we shall support all released LabVIEWs >= 2020 SP1
- Confirmed with both code observation and hand-testing that the .vip files install into LV 2022 Q3 (.. and measurements work therein).

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -28,33 +28,31 @@ The MeasurementLink Support for LabVIEW packages enable measurement developers t
...
+- InstrumentStudio 2023 Q1 or newer
+- MeasurementLink 2023 Q1 or newer
 - LabVIEW 2020 SP1 or newer
-- InstrumentStudio - 2023 Q1 or newer
-- MeasurementLink - 2023 Q1 or newer
+  - VI Package Manager 2020 or newer
 
 ---
 
 ## Software support
```
Comment (dixonjoel) [inline docs/README.md:38]: This section just appears like a repeat of the previous. I suggest we combine them. Not sure what the header should be.

Comment (aepete) [reply docs/README.md:38]: Agree.  Maybe we should just remove the "Dependencies" section.  We already cover version compatibility in "Software support", and we follow that with "Installation", which lists everything.

Comment (aepete) [reply docs/README.md:38]: Update coming.

Comment (aepete) [reply docs/README.md:25]: ... and while I'm here, it looks like there are a few stale quick-links in the header.  I'll clean those up too.

**Pull request merged**